### PR TITLE
chore(session-analysis): add $session_duration property key info

### PR DESCRIPTION
**Pull request by rcmarron**
## Problem

Session duration properties didn't get cool formatting like other PostHog properties.

## Changes

Adds session duration to the property key info. A bit of hack, but I think it's worth it.

image

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Selected session duration property in the property filters + breakdowns. Verified it looks good.

### CODE REVIEW

File: frontend/src/scenes/insights/views/InsightsTable/InsightsTable.tsx [hunk 1]
```diff
@@ -252,7 +252,7 @@ export function InsightsTable({
                     />
                 ),
                 render: function RenderPeriod(_, item: IndexedTrendResult) {
-                    return item.action.math_property
+                    return item.action?.math_property
```
Comment (rcmarron) [inline frontend/src/scenes/insights/views/InsightsTable/InsightsTable.tsx:255]: Unrelated bug fix that was causing a crash on using formulas.

**Review by fuziontech — APPROVED** [review]
Looks like Typescript!
🚢 it
